wagtail/admin/action_menu.py

```diff
@@ -20,6 +20,7 @@ class ActionMenuItem(metaclass=MediaDefiningClass):
     label = ''
     name = None
     classname = ''
+    icon_name = ''
```

Comment:
This should be added to the documentation at https://github.com/wagtail/wagtail/blob/master/docs/reference/hooks.rst#register_page_action_menu_item